File: day5/day5es1.js
function calcolaFattoriale(n) {
  if (n < 0) {
    return "Non è possibile calcolare il fattoriale di un numero negativo.";
  }
  
  let risultato = 1;
  for (let i = 1; i <= n; i++) {
    risultato *= i;
  }
  return risultato;
}


console.log(calcolaFattoriale(5)); 

File: day5/day5es1.py
def calcola_fattoriale(n):
    if n < 0:
        return "Non è possibile calcolare il fattoriale di un numero negativo."
    
    risultato = 1
    for i in range(1, n + 1):
        risultato = risultato * i
    return risultato


print(calcola_fattoriale(5))  